src/helper.js

```diff
@@ -43,3 +48,66 @@ export const onEventChange = (queryObject, key, event, idx) => {
 
   return { queryObject, updatedRules };
 };
+
+const validateName = (inputString) => {
+  const reg = /^[\p{L}]+$/u;
+  const isValid = reg.test(inputString);
+
+  return isValid;
+};
+
+const validateAge = (inputString) => {
+  const reg = /^(?:[0-9]|[1-9][0-9]|1[0-2][0-9]|130)$/;
+  const isValid = reg.test(inputString);
+
+  return isValid;
+};
+
+const validateLevel = (inputString) => {
+  const reg = /^[0-9]{1,6}$/;
+  const isValid = reg.test(inputString);
+
+  return isValid;
+};
+
+const validateEnrollmentYear = (inputString) => {
+  const reg = /^(?!0)[0-9]{4}$/;
+  const isValid = reg.test(inputString);
+
+  return isValid;
+};
+
+export const validateInput = (queryObject, event, idx) => {
+  let isValid = true;
+  let errorMessage = '';
+
+  queryObject.rules.forEach((rule) => {
+    if (rule.id === idx) {
+      const currentRule = rule;
+      if (
+        currentRule.field === 'First Name' ||
+        currentRule.field === 'Last Name'
+      ) {
+        isValid = validateName(event);
+        errorMessage = 'Invalid name';
```

Comment:
Use a more informative error message here, perhaps something like 'Name should be a string', so that the user knows why the name is invalid.